clang/lib/CIR/Dialect/IR/CIRTypes.cpp

```diff
@@ -369,6 +369,22 @@ BoolType::getABIAlignment(const ::mlir::DataLayout &dataLayout,
   return 1;
 }
 
+//===----------------------------------------------------------------------===//
+//  Definitions
+//===----------------------------------------------------------------------===//
+
+llvm::TypeSize
+ArrayType::getTypeSizeInBits(const ::mlir::DataLayout &dataLayout,
```

Comment:
Are these functions being called with any of the test cases? I don't see any checks that would be related to this.

Reply:
I was thinking of `sizeof` operator but it's not implemented yet, is there is another way to test the size in lit test

Reply:
My point was that if the code isn't being called, you should leave it out until it is needed.

Reply:
Yes, but it's part of the `DataLayoutTypeInterface` interface so we must implement it, I can test it after merging the sizeof PR https://github.com/llvm/llvm-project/pull/130847

Reply:
I see. Thanks for the clarification.